src/lib/chainNode.ts

```diff
@@ -31,18 +31,13 @@ export interface ProcessRanEvent {
   outputs: number[]
 }
 
-interface RoleEnum {
-  name: string | undefined
-  index: number | undefined
-}
-
 interface SubstrateToken {
   id: number
   metadata: {
     [key in string]: { literal: string } | { file: string } | { tokenId: number } | { None: null }
   }
   roles: {
-    [key in 'Owner' | 'MemberA' | 'MemberB' | 'Optimiser']: string
+    [key in 'hydrogen_owner' | 'energy_owner' | 'owner']: string
```

Comment:
I thought we are supposed to only add 'hydrogen_owner' and 'energy_owner' without the a third one .. ?!

Reply:
Owner, has been removed